package.json

```diff
@@ -20,15 +20,16 @@
   "homepage": "https://github.com/zazuko/rdf-loader-code",
   "dependencies": {
     "@rdfjs/namespace": "^1.1.0",
-    "clownface": "^0.12.0"
+    "clownface": "^1.0.0"
   },
   "devDependencies": {
-    "@rdfjs/data-model": "^1.1.2",
+    "@rdfjs/data-model": "^1.2.0",
     "@rdfjs/dataset": "^1.0.1",
-    "@rdfjs/parser-n3": "^1.1.2",
-    "jest": "^24.9.0",
+    "@rdfjs/parser-n3": "^1.1.4",
+    "c8": "^7.3.1",
+    "codecov": "^3.7.2",
```

Comment:
Slightly unrelated, I typically don't install this as dependency but rathe call `npx codecov` in CI